.dockerignore

```diff
@@ -1,5 +1,3 @@
-# we want to get the git commit SHA but not all the binary repo data
-.git/objects/pack
```

Comment:
this will make the container build really long if this container is the primary one used for compiling, the history for netplugin includes binaries, etc. and is copied over

Reply:
yes, that's intended, without objects file git couldn't figure out which version it's on, thus will put the version info wrong